src/network/tc/etf.h

```diff
@@ -0,0 +1,37 @@
+/* SPDX-License-Identifier: LGPL-2.1+
+ * Copyright © 2020 Intel Corporation */
```

Comment:
we usually don't mark files with 'copyright', as it's not really useful (also 2020 is a bit demode')

Reply:
Thanks i will remove that on my next spin.